gapic/api/artman_longrunning.yaml

```diff
@@ -1,5 +1,5 @@
 common:
-  api_name: google-longrunning
+  api_name: google-longrunning-v1
```

Comment:
I'm quite hesitant to use v1 here, since "v1" technically isn't taken for it yet. What if someone names the next major version of it "v1"?